core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -200,5 +244,62 @@ var ExplorationEditorTranslationTab = function() {
     expect(element(by.css('.protractor-test-translation-feedback-tab'))[0]
     ).toEqual(element(by.css('.oppia-active-translation-tab'))[0]);
   };
+
+  this.openUploadAudioModal = function() {
+    waitFor.elementToBeClickable(
+      uploadAudioButton, 'Upload Audio button is not clickable');
+    uploadAudioButton.click();
+  };
+
+  var translationGraph = element(by.css('.protractor-test-translation-graph'));
+  var stateNodes = translationGraph.all(by.css('.protractor-test-node'));
+  var stateBackgroundNodes = translationGraph.all(by.css(
+    '.protractor-test-node-background'));
+  var stateNodeLabel = function(nodeElement) {
+    return nodeElement.element(by.css('.protractor-test-node-label'));
+  };
+
+  // NOTE: if the state is not visible in the state graph this function will
+  // fail.
+  this.moveToState = function(targetName) {
+    general.scrollToTop();
+    stateNodes.map(function(stateElement) {
+      return stateNodeLabel(stateElement).getText();
+    }).then(function(listOfNames) {
+      var matched = false;
+      for (var i = 0; i < listOfNames.length; i++) {
+        if (listOfNames[i] === targetName) {
+          stateNodes.get(i).click();
+          matched = true;
+          // Wait to re-load the entire state editor.
```

Comment:
Add a proper comment.

Reply:
Removed the comment as it was unnecessary.